service.py

```diff
@@ -37,9 +38,69 @@ def autostart():
         else:
             cleaner.monitor.waitForAbort(service_sleep)
 
-    log("Abort requested. Terminating.")
+    debug("Abort requested. Terminating.")
```

Comment:
Well spotted. I apparently forgot to change all ``log()``s back to ``debug()``s.